app/tests/test_serializer.py

```diff
@@ -800,6 +800,9 @@ def test_asset_serialization(self):
         # translate to Python native:
         serializer = AssetSerializer(self.asset, context={'request': request})
         python_native = serializer.data
+        # hack to deal with the item property, as it is "write_only", it will not appear
+        # in the mocked request's data. So we manually add it here:
+        python_native["item"] = item_name
```

Comment:
I would move this to the `python_native_back` below because this hack is only for de-serialization needed and not for serialization.

Reply:
Makes sense, done